extensions/wikia/Search/js/WikiaSearch.js

```diff
@@ -81,8 +89,103 @@ var WikiaSearch = {
 			searchForm.submit();
 		});
 
+	},
+	trackSearchResultClick: function(clickedElement) {
+		var queryparams = new URL(window.location).searchParams;
+		var query = queryparams.get('search') || queryparams.get('query');
+
+		if (!query) {
+			return;
+		}
+
+		var payload = {
+			searchPhrase: query,
+			clicked: {
+				type: 'article', // we don't show wikis results right now
+				id: parseInt(clickedElement.getAttribute('data-page-id')),
```

Comment:
Do we need to cast these values to ints?

Reply:
yes, this is requirement from data-eng

Reply:
👍